openreview/venue_request/process/deployProcess.py

```diff
@@ -78,6 +78,8 @@ def process(client, note, invitation):
 
 Thank you for choosing OpenReview to host your upcoming venue.
 
+Users of new profiles that sign up with non institutional emails will go through a moderation process that **can take up to two weeks**. Users that signup for a profile with their institutional email will be activated automatically. 
```

Comment:
Sounds great! I think we should suggest the PCs to share this information in the "Calls for papers". @celestemartinez do you have any suggestions for the wording?

Reply:
"To encourage early profile registration, we recommend alerting authors of this extended moderation period in the Call for Papers."

How about something like this?

Reply:
The bullet points might be easier to read. I'll make the change.